src/core/reactor.cc

```diff
@@ -4822,6 +4822,14 @@ scheduling_group::set_shares(float shares) noexcept {
     }
 }
 
+future<> scheduling_group::update_io_bandwidth(uint64_t bandwidth) const {
+    if (!engine().use_scheduling_groups_as_io_classes()) {
+        return make_exception_future<>(std::runtime_error("Bandwidth should be updated directly on io_priority_class (deprecated)"));
+    }
+
+    return internal::make_priority_class_from_sched_group(*this).update_bandwidth(bandwidth);
```

Comment:
In fact it's creating a new object, because io_priority_class is handle-like one that only contains an ID and can be copied multiple times. But this comment is no longer relevant :( Avi suggested that unification is done compile-time, so I've a new branch where this function doesn't exist